app/api_key.py

```diff
@@ -0,0 +1 @@
+API_KEY = "9c4defd91fc0412ba8e144724241611"
```

Comment:
Storing API keys directly in the source code is a security risk. Consider using environment variables or a secure vault to manage and access sensitive information like API keys.